lib/api/objectDelete.js

```diff
@@ -22,10 +22,6 @@ const versionIdUtils = versioning.VersionID;
  */
 function objectDelete(authInfo, request, log, cb) {
     log.debug('processing request', { method: 'objectDelete' });
-    if (authInfo.isRequesterPublicUser()) {
-        log.debug('operation not available for public user');
-        return cb(errors.AccessDenied);
-    }
```

Comment:
should this be addressed in a different ticket (to make cloudserver more like aws)? it seems to be out of scope of the fix, and I believe this may be something that would warrant further discussion.

Reply:
would be undoing related commit that has more to it: https://github.com/scality/cloudserver/commit/12b866dc680df403bb771a59426e9e9e18b101bd

Reply:
Those 4ish checks seem like things that should be checked in the ACL stuff tbh. I can undo that work and tweak the tests so that the delete does not work anymore and save the rest for later.